src/routes/transactions/transactions.controller.ts

```diff
@@ -233,6 +234,7 @@ export class TransactionsController {
   @Get('chains/:chainId/safes/:safeAddress/transactions/history')
   @ApiQuery({ name: 'timezone_offset', required: false, type: String })
```

Comment:
Perhaps we should mark this as deprecated. What do you think?

Reply:
Done ✅